docker-compose.yml

```diff
@@ -79,16 +98,17 @@ services:
       #- MAIL_FROM_NAME=
       #- MAIL_FROM_ADDRESS=
       #- TRUSTED_PROXIES=
-      #- SKIP_PERMISSIONS_CHECKS
+      - SKIP_PERMISSIONS_CHECKS=${SKIP_PERMISSIONS_CHECKS:-false}
       - STARTUP_DELAY=30
       #- ADMIN_USER=admin
       #- ADMIN_PASSWORD=<ADMIN_PASSWORD>
       #- ADMIN_PASSWORD_FILE=<filename>
-      ### Unused in Lychee
-      #- REDIS_HOST=127.0.0.1
-      #- REDIS_PASSWORD=null
-      #- REDIS_PASSWORD_FILE=<filename>
-      #- REDIS_PORT=6379
+      - CACHE_DRIVER=${CACHE_DRIVER:-redis}
+      - REDIS_URL=redis://${REDIS_USERNAME:-default}:${REDIS_PASSWORD:-}@${REDIS_HOST:-lychee_redis}:${REDIS_PORT:-6379}
+      - REDIS_HOST=${REDIS_HOST:-lychee_redis}
+      - REDIS_PORT=${REDIS_PORT:-6379}
+      - REDIS_PASSWORD=${REDIS_PASSWORD:-}
+      - LOG_VIEWER_CACHE_DRIVER=${LOG_VIEWER_CACHE_DRIVER:-file}
```

Comment:
This one's also not in Lychee's `.env.example` :man_shrugging:

Reply:
Done: https://github.com/LycheeOrg/Lychee/pull/3003